src/fec_base.h

```diff
@@ -755,6 +769,75 @@ void FecCode<T>::decode_lagrange(
         output->set(i, S.get(i));
 }
 
+template <typename T>
+void FecCode<T>::decode_vec_lagrange(
+    vec::Vector<T>* output,
+    const std::vector<Properties>& props,
+    off_t offset,
+    vec::Vector<T>* fragments_ids,
+    vec::Vector<T>* words,
+    vec::Vector<T>* vx,
```

Comment:
can be const? or even a const reference (if nullptr is not allowed).
Same remarks for `words` and `fragments_ids` above.